qiskit/transpiler/passmanager_config.py

```diff
@@ -113,5 +116,11 @@ def from_backend(cls, backend, **pass_manager_options):
             res.instruction_durations = InstructionDurations.from_backend(backend)
         if res.backend_properties is None:
             res.backend_properties = backend.properties()
+        if res.target is None:
+            backend_version = getattr(backend, "version", 0)
+            if not isinstance(backend_version, int):
+                backend_version = 0
+            if backend_version >= 2:
+                res.target = backend.target
```

Comment:
`target` is a exact-v2 feature. While the structure of v3 is unknown, I think this would be better:

```suggestion
            if backend_version == 2:
                res.target = backend.target
            if backend_version > 2:
                raise TranspilerError(f"Backend version {backend_version} not supported.")
```